hyperspy/signal.py

```diff
@@ -1928,7 +1928,7 @@ def __call__(self, axes_manager=None):
             self.data.__getitem__(axes_manager._getitem_tuple))
 
     def plot(self, navigator="auto", axes_manager=None,
-             plot_markers=True, **kwargs):
+             convert_units=True, plot_markers=True, **kwargs):
```

Comment:
Actually, if the units are converted during loading, there is no point to set `convert_units=True` as default.